### ISSUE TITLE
Fibaro NEW FGSD-002 Smoke Sensor not reconized on Homey

### ISSUE THREAD

**Issue by Handyman4u** [athombv/homey#2477]
**Metadata**

* **Homey Version**: Homey Firmware Versie: 1.5.11
* **Homey Log Code**:
* **Smartphone Platform**:
* **Smartphone App Version**:

**If related to a Homey App:**
* **Homey App Version**:v1.5.17
* **Homey App Log Code**: cfbb71e3-324b-47b0-be7b-4dfa9b69a00e
* **Affected Devices (preferably product codes)**: FGSD-002 ZW5 v3.3

**What did you try to achieve?**
trying to add an new Fibaro Smoke sensor FGSD-002 ZW5 v3.3 Build April 2018

**What did you expect as result?**
reconizion of an Fibaro Smoke detector on the Homey

**What was the result?**
New Fibaro Smoke sensor is not reconized on the homey and will result in an Unknown Z-wave device

**Any other remarks**

tryed 2 new Fibaro Smoke detectors (April 2018) on 2 diffrent Homey's with same results.

**Comment by bvdbos** [comment]
Athom uses GitHub for issue tracking in existing software only. For new devices they use https://go.athom.com/request-device-support

**Comment by 2benl** [comment]
Well there is an Fibaro app already and one of Homey’s employees is rebuilding this app to V2. Maybe he can also add this or aks Caseda on Slack. Or make a Github issue in the fibaro github.

**Comment by jeroenvollenbrock** [comment]
Hi! Thank you for requesting support for a device.

We use GitHub for issue tracking in existing software only. Could you please fill out this form to request support for your device? We will then consider to support it! https://go.athom.com/request-device-support

I am closing this issue now. Please continue to report any bugs you may encounter in the future here.

**Comment by razorfish79** [comment]
@jeroenvollenbrock

I guess this is not a feature request as this smoke sensor is implemented in the Fibaro app. Homey just does not recognize it correctly. This is how I understand this as the sensor and its firmware is not a new fibaro device.

**Comment by Handyman4u** [comment]
agreed, and now? just closing this ticket is not very helpfull, just seen you have also closed the one on 
athombv/com.fibaro #230
so whats the next step now, somebody working on this to get it resolved?

**Comment by jeroenvollenbrock** [comment]
Oh excuse me, the word NEW led me to believe it was actually a new product that we did not support yet. Could you pair it as an unknown device, go to the device settings, and post us a screenshot of the node information tab? (It's collapsed by default).

**Comment by Handyman4u** [comment]
allright, 
here you go:
![fgsd-002 zw5 v3 3](https://user-images.githubusercontent.com/39384777/40810855-62a66dc4-652f-11e8-9f86-256b5a191640.png)

**Comment by RWensveen** [comment]
just a quick remark: I have several Fibaro smokesensors FGSD-002, all working well with Homey.
There is however 1 difference in de node-information: my device has 'Product ID' = 4098, your has 4099.

**Comment by Handyman4u** [comment]
can u check which build it is, mine is from: FGSD-002 ZW5 v3.3 Build April 2018, so brand new.
i have treid 2 smoke sensors on 2 homey's with same result.

**Comment by RWensveen** [comment]
I don't have the box it came in, but on the back of the device, it says FGSD-002, EU, 04 Aug 2017.
it has the same 'firmware version' as in the picture above

**Comment by BirdieVia** [comment]
I do have the same problem, same sensor build and fw version, FGSD-002 ZW5 v3.3 Build April 2018, as Handyman4u. Sensor is detected as Unknown Z-wave device.

**Comment by MatthewWaanders** [comment]
Hi guys!

This is indeed a problem with the ProductID, when the SDK2 version is fully merged together I'll do a run over all Fibaro app issues like this one to add the missing productIDs, so they should all work when that update comes out.

Until then, changes in our SDK will not allow me to live update the Fibaro app, so I'll have to ask you guys to be patient for just a little longer!

**Comment by Handyman4u** [comment]
Hi Matthew,
Some time scale would be nice are we talking about day's, weeks or month's

**Comment by MatthewWaanders** [comment]
Hi guys!

As you know I can't provide any set timelines, this is a big app with a large user base. I can however tell you guys that all devices are merged together now, and testing and patching has begun. I'm not sure how long this process will take though, I hope you'll understand.

**Comment by gvrsnor** [comment]
A Days/weeks/months/V2.0 indication would be nice. I want to buy some fibaro sensors but if I can't use them yet..

**Comment by Reemface** [comment]
Same issue here, with three FGSD-002 sensors.

Model: FGSD-002 ZW5 v3.3
Date: 08-01-2018, 08-05-2018
EAN: 5902020528265
Fabrikant: 271
Product Type ID: 3074
Product ID: 4099
Firmware ID: 3090

Homey Firmware Versie: 1.5.11
Fibaro App: v1.5.18

**Comment by Sutcivni** [comment]
Have the same issue. Hopefully a fix will be in place soon :)

**Comment by Handyman4u** [comment]
Well I really doubt this...
According to the last mail van Homey it seems that they are testing the 1.5.13 release shortly to beta, my guess is another month for this release and then after this the V2 will be tested & released, and this will take more time again..... that’s why they can't tell any time scale, my guess is that we looking at least 2 month's for the new V2 release, which should fix this issue which is fixable according to a different post:
The product ID needs to be updated:

Adding the 4099 to the product id solves the problem. I send a pull request, but can be run manually by uninstalling the Fibaro drivers, then,

npm install -g athom-cli
athom login
git clone http://github.com/mlund/com.fibaro
athom app install ./com.fibaro

but unsure how to do this and which app i need to runs this commands on the homey

Any help would be appreciated

**Comment by Handyman4u** [comment]
Thx,

Allright i installed the app and took the zip file from: 
https://github.com/athombv/com.fibaro/archive/master.zip, since git clone from above is not working...
but oke orignal App is installing ok, and app installed as develepment app,  but since the pruduct ID is not updated this is still not working...

The zip file from: http://github.com/mlund/com.fibaro
(https://github.com/mlund/com.fibaro/archive/master.zip) does not work, when installing the app gives this result:  Pre-processing app...
Could not find a valid Homey App at `C:\Homey\com.fibaro-master`

so looks like this is not working,....

**Comment by mruiter** [comment]
its clear youre doing something wrong big time.
since you hav a zip it seems you didnt even clone the git right.
but start with unpacking the zip. dont forget to put all the needed node libs like zwave mesh driver into it 2

**Comment by bvdbos** [comment]
Just installed the zip you linked, installled through the cli without a
problem, no extra modules needed but you need to unzip the file like Marco
says.

**Comment by Fusionswe** [comment]
and did it add the missing hardwareid after the installation? :)

Skickat från min iPhone

**Comment by Handyman4u** [comment]
I did unzip but but the unzipped files where in:

C:\Homey\com.fibaro-master\com.fibaro-master, so i had to go to this folder offcourse :-(
but now indeed succesfully installed!

added my smoke sensor, but still the same, Unknow Zwave device...
Looking at the code from Mlund, only the Floodsensor product ID was added...
should this be enough for the Smoke sensor product ID to, or should this be added seperatly.

Anyway thanks for the help  so far, getting there slowly...

**Comment by Fusionswe** [comment]
I edited the file app.json and added the missing productID. "4099"
Then i removed the unknown z-wave devices and added the smokedetectors without any problems.
And hopefully they work =)

---
"id": "FGSD-002",
      "name": {
        "en": "Smoke Sensor (Z-Wave Plus)",
        "nl": "Rookmelder (Z-Wave Plus)"
      },
      "zwave": {
        "manufacturerId": 271,
        "productTypeId": 3074,
        "productId": [
          4099,
          4098,
          8194,
          12290
-----

**Comment by Handyman4u** [comment]
yep did the same now and working:

![afbeelding](https://user-images.githubusercontent.com/39384777/45380394-e0a68780-b602-11e8-8435-7651d5c116ff.png)

Hoera finally solved!

Thanks for the support guy's

**Comment by Reemface** [comment]
This issue is solved for me by doing:
- update homey firmware to 1.5.12
- update fibaro app to beta 2.1.0

Now i can pair all three smoke detectors.
Thanks!!

**Comment by Veldkornet** [comment]
Is the tamper alarm supposed to work in the beta app (v2.1.8) ? Or is it still missing?

I haven't tried to re-pair the smoke detector yet, I first wanted to check whether it's possible yet or not.
